app/service/data_svc.py

```diff
@@ -390,6 +390,22 @@ def __init__(self, dao):
         :return: None
         """
         await self.dao.update(table, key, value, data)
+        
+    async def update_agent(self, table, data):
```

Comment:
seems this function is very specific to agent executors and may be to say update_agent_executors and drop the table input